### DISCUSSION TITLE
When will coc.nvim release a new version ?

### DISCUSSION THREAD

**Discussion by Weissle** [neoclide/coc.nvim#3827] [General]
Noticing that it is been a year and a half since the last release, so I am wondering when will coc.nvim have a new release ?

**Comment by chemzqm** [comment]
Should be soon, the only left feature of LSP 3.1.6 is annotations and transaction support of `WorkspaceEdit`

**Reply by Weissle** [reply]
Get it. I am looking forward to it.
